clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerConfig.java

```diff
@@ -172,6 +172,10 @@
     public static final String MAX_POLL_RECORDS_CONFIG = "max.poll.records";
     private static final String MAX_POLL_RECORDS_DOC = "The maximum number of records returned in a single call to poll().";
 
+    /** <code>exclude.internal.topics</code> */
+    public static final String EXCLUDE_INTERNAL_TOPICS_CONFIG = "exclude.internal.topics";
+    private static final String EXCLUDE_INTERNAL_TOPICS_DOC = "Whether messages from internal topics (such as offsets) should be exposed to the consumer.";
```

Comment:
@hachikuji I actually copied the description from the old consumer config ([here](http://kafka.apache.org/documentation.html#oldconsumerconfigs)) to remain consistent. If "true" internal topics will not be exposed to the new consumer; if "false" they will.

With respect to listTopic() API nothing is changed, which is, if I'm not mistaken, internal topics are still returned.